docs/source/config_file.rst

```diff
@@ -230,10 +230,15 @@ section of the command line docs.
 
     This option may only be set in the global section (``[mypy]``).
 
+``no_site_packages`` (bool, default False)
+    Disables searching for :pep:`561` compliant packages. This will also
```

Comment:
Regular users won't recognize most PEP numbers, so it would be better to give a short summary of what this means here, as otherwise most readers would have to open the link to understand what is being talked about. Maybe something like "Disables using type information in installed packages (PEP 561)".